packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -125,11 +122,10 @@ class TransactionService {
        */
       if (__DEV__) {
         this._logger.debug(
-          'Redefining the current transaction',
-          tr,
-          name,
-          type,
-          perfOptions
+          'Redefining current transaction from',
+          { name: tr.name, type: tr.type },
+          'to',
+          { name, type, options: perfOptions }
```

Comment:
I think here having the full transaction helps. But you can instead include name and type in the message and pass the transaction as the last argument

Reply:
Sure, so remove options and instead add transaction instead?